modules/twinklewarn.js

```diff
@@ -16,6 +16,9 @@
 Twinkle.warn = function twinklewarn() {
 	if( mw.config.get( 'wgRelevantUserName' ) ) {
 			Twinkle.addPortletLink( Twinkle.warn.callback, "Warn", "tw-warn", "Warn/notify user" );
+			if (Twinkle.getPref('autoMenuAfterRollback') && mw.config.get('wgNamespaceNumber') === 3 && mw.util.getParamValue('vanarticle')) {
```

Comment:
I feel like there's a better/more twinkley way of doing this...

Reply:
This approach seems fine to me.